netlify.toml

```diff
@@ -0,0 +1,17 @@
+[build]
```

Comment:
Please add `netlify.toml` under zuul job irrelevant-files list here: https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/zuul.d/jobs.yaml#L11 so that zuul jobs does not trigger on this change.